perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -752,6 +752,8 @@ def Prepare(self, vm):
                      '--rc-path=.bash_profile '
                      '--path-update=true '
                      '--bash-completion=true')
+    # Need to install crcmod for gcp. See "gsutil help crc" for details.
+    vm.RemoteCommand('sudo pip install -U crcmod')
```

Comment:
How about checking to see if it is installed first and logging.  We should also differentiate the default experience from the improved experience by adding meta data on when you have to stall crcmod.

How do you plan to do that?